airbase/pom.xml

```diff
@@ -512,7 +513,6 @@
 
                             <!-- Pass the following three directly to the JVM,
```

Comment:
"three" ?

Reply:
Hah... Proof that nobody looks at comments (except a reviewer, sometimes) :) Will change

Reply:
Done

Reply:
> if you feel no-one is going to read it, why would we keep it at all?

I was just being snarky, sorry. After X years in this business my brain apparently got trained to filter out comments and I just don't see them anymore, unless I make an effort.